src/Core/src/Handlers/Element/IRemappable.cs

```diff
@@ -0,0 +1,6 @@
+namespace Microsoft.Maui.Handlers;
+
+internal interface IRemappable
+{
+	void RemapForControls();
```

Comment:
This can be static:

https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IPlatformApplication.cs#L18

Sure, it won't work for net standard, but that is an unsupported platform and is just there for old VS things. Not even sure hot reload uses it.